eo-parser/src/test/java/org/eolang/parser/XMIRTest.java

```diff
@@ -83,12 +84,12 @@ void printsToEO(final String src) throws Exception {
      * @throws IOException If fails
      */
     private static XML parse(final String source) throws IOException {
-        return new XSLDocument(
-            XMIRTest.class.getResourceAsStream("wrap-method-calls.xsl")
-        ).with(new ClasspathSources()).transform(
-            new EoSyntax(
-                "test", new InputOf(source)
-            ).parsed()
+        return new Xsline(new TrClasspath<>(
```

Comment:
@maxonfjvipon I'm curious, how does this line pass Qulice check? There is obviously an error. Maybe we have Qulice turned off for this module?

Reply:
@yegor256 no, we don't, seems like a bug in Qulice. And yes, there's something wrong with indentations. @c71n93 please move `new TrClasspath<>(` to the next line